<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserConection</class>
 <widget class="QWidget" name="UserConection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_password">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Contraseña</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_url">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhUrlCharactersOnly</set>
   </property>
   <property name="placeholderText">
    <string>Url webservice</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_user">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Uusuario</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_conn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Conexión</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>on_connection_clicked()</slot>
 </slots>
</ui>
